shared/js/ui/views/tracker-networks.es6.js

```diff
@@ -77,9 +77,10 @@ TrackerNetworks.prototype = window.$.extend({},
 
         _rerender: function (e) {
             if (e && e.change) {
-                if (e.change.attribute === 'isPartOfMajorTrackingNetwork' ||
-            e.change.attribute === 'isWhitelisted' ||
-            e.change.attribute === 'totalTrackerNetworksCount') {
+                if (e.change.attribute === 'isaMajorTrackingNetwork' ||
+                    e.change.attribute === 'isWhitelisted' ||
+                    e.change.attribute === 'totalTrackerNetworksCount' ||
+                    e.change.attribute === 'siteRating') {
```

Comment:
@MariagraziaAlastra this is the line that resolves the green tracker networks icon issue. There was a race condition which meant the `siteRating` was updated after the view had already rendered, which meant the site became D and didn't get reflected in the tracker networks icon.

I'm not sure why we only just saw it in this PR. Possibly because the threshold for a site being "D" is slightly different.

Reply:
Cool, looks fixed now, thanks!